server/stats.js

```diff
@@ -4,9 +4,8 @@ const _ = require('underscore');
 const monk = require('monk');
 
 const GameService = require('./services/GameService.js');
-const config = require('./config.js');
 
-let db = monk(config.dbPath);
+let db = monk('mongodb://127.0.0.1:27017/throneteki');
```

Comment:
Changes in this file seem unrelated (perhaps stuff you've done for local dev?)

Reply:
Kind of unrelated, i guess a seperate PR is a good idea. I would like an addition to the stats script to get the number of unique players (see line 139 to 142) and this change is necessary as the config.js file got removed some time ago

Reply:
Will revert these changes and open up a seperate PR with the usage of configservice, thanks!